core/templates/pages/topic-viewer-page/topic-story-section/topic-story-section.component.css

```
@@ -0,0 +1,278 @@
+  position: relative;
+  z-index: 2;
+}
+
+.story-card-header-left {
+  flex: 1 1 auto;
+  min-width: 0;
+}
+
+.story-title {
+  color: #fff;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Capriola" (font-family-name-quotes)

(font-family-name-quotes)